_topic_map.yml

```diff
@@ -1277,16 +1277,12 @@ Topics:
     File: working-with-pipelines-using-the-developer-perspective
   - Name: OpenShift Pipelines release notes
     File: op-release-notes
-#- Name: GitOps
-#  Dir: gitops
-#  Distros: openshift-enterprise,openshift-webscale,openshift-origin
-#  Topics:
-#  - Name: Understanding OpenShift GitOps
-#    File: understanding-openshift-gitops
-#  - Name: Installing OpenShift GitOps Operator
-#    File: installing-openshift-gitops-operator
-#  - Name: Getting started with OpenShift GitOps
-#    File: getting-satrted-with-openshift-gitops
+- Name: GitOps
+  Dir: gitops
+  Distros: openshift-enterprise,openshift-webscale,openshift-origin
+  Topics:
+  - Name: Working with OpenShift GitOps
```

Comment:
Retain the first and third assemblies. Remove "installing openshift..." remove "working with..."
add "getting started..." and "understanding.."